hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -663,6 +663,16 @@ public class HoodieWriteConfig extends HoodieConfig {
           + "will not print any configuration which contains the configured filter. For example with "
           + "a configured filter `ssl`, value for config `ssl.trustore.location` would be masked.");
 
+  public static final ConfigProperty<Boolean> ROLLBACK_BACKUP_ENABLED = ConfigProperty
+      .key("hoodie.rollback.backup.enabled")
+      .defaultValue(false)
+      .withDocumentation("Backup instants removed during rollback and restore (useful for debugging)");
+
+  public static final ConfigProperty<String> ROLLBACK_BACKUP_DIRECTORY = ConfigProperty
+      .key("hoodie.rollback.backup.dir")
```

Comment:
hoodie.rollback.timeline.backup.dir

Reply:
Renamed to hoodie.rollback.instant.backup.dir